processor/metricstransformprocessor/operation_scale_value.go

```diff
@@ -43,3 +47,18 @@ func scaleValueOp(metric pmetric.Metric, op internalOperation, f internalFilter)
 		}
 	}
 }
+
+func scaleHistogramOp(metric pmetric.Metric, op internalOperation, f internalFilter) {
+	var dps = metric.Histogram().DataPoints()
+
+	for i := 0; i < dps.Len(); i++ {
+		dp := dps.At(i)
+		if !f.matchAttrs(dp.Attributes()) {
+			continue
+		}
+		dp.SetSum(dp.Sum() * op.configOperation.Scale)
+		for bounds, bi := dp.ExplicitBounds(), 0; bi < bounds.Len(); bi++ {
+			bounds.SetAt(bi, bounds.At(bi)*op.configOperation.Scale)
+		}
```

Comment:
Shouldn't we also scale min, max and exemplars values if present?

Reply:
We should, yes, thanks. I'll add it.